dev/dev-env/dev-shell.nix

```diff
@@ -0,0 +1,15 @@
+pkgs:
+
+pkgs.mkShell {
```

Comment:
it would be more logical for me to put this file in `/dev` because it has nothing to do with the "dev-env" that's created with `create.sh`.

Reply:
Done.